apps/biobin/Readme.developer.md

```diff
@@ -1,14 +1,47 @@
-This app dynamically fetches some resources from other public projects:
+# Biobin
 
+## What does this app do?
+This app runs the BioBin software, which is a comprehensive bioinformatics tool for biologically-driven binning and association analysis of rare variants (RVs) in sequence data. It automates multi-level binning or collapsing of RVs into biological features such as genes, pathways, regulatory regions, evolutionary conserved regions (ECRs), and many others. BioBin improves on other variant binning algorithms through the use of prior biological information, which can highlight the potential cumulative effects of biologically aggregated RVs. BioBin requires the Library of Knowledge Integration (LOKI), which contains diverse biological knowledge from multiple collections of publically available databases. 
 
-Ritchie Lab Software:/BioBin/biobin
-	The DNAnexus-compiled binary of BioBin.
+## Where can I find the BioBin software manual?
+You can find the detailed BioBin software manual here: http://www.ritchielab.com/software/biobin-download
 
-Ritchie Lab Software:/BioBin/biobin-summary.py
-	A helper script which collates and summarizes BioBin output files.
+## What data is required for this app to run?
+This app requires a VCF file to run. If the user has a VCF TBI Index file available, he/she can upload that as an input as well. Otherwise the app will generate the index file for the inputted VCF file. Depending on the type of analysis that the user would like to run (e.g. limiting the analysis to a list of desired genes or to just LOF variants etc.), there are additional input files that need to be included (for more information on which input files to use and on the format of input files, please refer to the software manual above):
+  - Phenotype file (`*.phe`)
+  - Covariate file (`*.cov`)
+  - Role file (`*.rol`)
+  - Weight file (`*.wgt`)
+  - Custom regions file (`*.rgn`) - referred to as "Region file" in the BioBin software manual
+  - Region names list (`*.gen`) - list of gene symbols (1 per line) that the user would like to limit the analysis to. This input file is specific to the app, it is not available in the BioBin command-line application.
+  - BioBin Binary Executable - used only if the user would like to use older versions of the app, should be ignored in most cases.
+  - BioBin Summary Script - used only if the user would like to use older versions of the script that generates the summary output file, should be ignored in most cases.
```

Comment:
- BioBin Summary Script - used only if the user would like to use older versions of the script that generates the summary output file; should be ignored in most cases